### PR TITLE
Update exportCypher.adoc

### PR DESCRIPTION
**Pull request by webtic**
added docs for the cypherFormat

### CODE REVIEW

File: docs/exportCypher.adoc [hunk 1]
```diff
@@ -12,12 +12,17 @@ To change the export format, you have to set it on the config params like `{form
...
+To change the cypher export format, you have to set it on the config params like `{cypherFormat: "updateAll"}`
 
 * "create": all CREATE
 * "updateAll": MERGE instead of CREATE
 * "addStructure": MATCH for nodes + MERGE for rels
 * "updateStructure": MERGE + MATCH for nodes and rels
 
+Format and cypherFormat can be used both in the same query giving you complete control over the exact export format:
+
+`call apoc.export.cypher.query("MATCH (e)","/tmp/theExportFile", {format:'plain',cypherFormat:'updateStructure'})`
```
Comment (jexp) [inline docs/exportCypher.adoc]: can you change this to a block format?

**Comment by webtic** [comment]
better now?

**Comment by jexp** [comment]
Updated the bit myself :) Thanks a lot

**Comment by webtic** [comment]
ah _thats_ a blockformat ;-)

**Comment by jexp** [comment]
Haha, no I just felt it's more readable that way.

**Pull request merged**